modules/serverless-logic-deploying-workflows-dev-mode.adoc

```diff
@@ -0,0 +1,137 @@
+// Module included in the following assemblies:
+//
+// * serverless/serverless-logic/serverless-logic-creating-managing-workflows.adoc
+
+:_content-type: PROCEDURE
+[id="serverless-logic-deploying-workflows-dev-mode"]
+= Deploying workflows in Dev mode
```

Comment:
Is "Dev mode" the official name or short for "Developer mode"?

Reply:
Yes, it is the official name for now.